docs/_source/getting_started/installation/configurations/user_management.md

```diff
@@ -8,23 +8,20 @@ Let's first describe Argilla's user management model:
 
 ### `User`
 
-A Argilla user is defined by the following fields:
+An Argilla user is defined by the following fields:
 
-- `username`: The username to use for login into the Webapp.
-- `email`(optional): The user's email.
-- `fullname` (optional): The user's full name
-- `disabled`(optional): Whether this use is enabled (and can interact with Argilla), this might be useful for disabling user access temporarily.
-- `workspaces`(optional): The team workspaces where the user has read and write access (both from the Webapp and the Python client). If this field is not defined the user will be a super-user and have access to all datasets in the instance. If this field is set to an empty list `[]` the user will only have access to her user workspace. Read more about workspaces and users below.
-- `api_key`: The API key to interact with Argilla API, mainly through the Python client but also via HTTP for advanced users.
+- `username`: The username used as login for Argilla's UI.
+- `first_name`: The user's first name.
+- `last_name` (optional): The user's last name.
+- `role`: The user's role in Argilla. Available roles are: "admin" and "annotator". Only "admin" users can create and delete workspaces and datasets, and change the dataset settings, like the labeling schema.
+- `workspaces`: The workspaces where the user has read and write access (both from the UI and the Python client). Read more about workspaces and users below.
+- `api_key`: The API key to interact with Argilla API, mainly through the Python client but also via HTTP for advanced users. It is automatically generated when a user is created.
 
 ### `Workspace`
 
-A workspace is a Argilla "space" where users can collaborate, both using the Webapp and the Python client. There are two types of workspace:
+A workspace is a "space" inside your Argilla instance where authorized users can collaborate. It is accessible through the UI and the Python client.
 
-- `Team workspace`: Where one or several users have read/write access.
-- `User workspace`: Every user gets its own user workspace. This workspace is the default workspace when users log and load data with the Python client. The name of this workspace corresponds to the username.
-
-A user is given access to a workspace by including the name of the workspace in the list of workspaces defined by the `workspaces` field. **Users with no defined workspaces field are super-users** and have access and right to all datasets.
+You can assign users to workspaces when you create a new user, or by using the proper API endpoint. "Admin" users can access to ALL defined workspaces
 
 
 ### Python client
```

Comment:
In line 29: The Python client gives developers the ability to log, load, and copy datasets from and to different workspaces.